docs/src/tasks.md

```diff
@@ -43,10 +43,12 @@ These variables allow you to pull information from the current editor and use it
 
 - `ZED_COLUMN`: current line column
 - `ZED_ROW`: current line row
-- `ZED_FILE`: absolute path to the file
+- `ZED_FILE`: absolute path of the currently opened file (e.g. `~/path/to/project/src/main.rs`)
```

Comment:
That's not an absolute path though, right? 😄 
```suggestion
- `ZED_FILE`: absolute path of the currently opened file (e.g. `/Users/my-user/path/to/project/src/main.rs`)
```

Reply:
Oops😄 Your right!

Reply:
I fixed both wrong absolute paths